features/resources/generated_responses_231/algod_GetGenesisResponse_0.json

```diff
@@ -17,38 +17,40 @@
       }
     },
     {
-      "addr": "LX6VRFCL4723LRO3VEX5XNOXHFXFQKATKDLDBT5HNT5DFDC3SQG4EI2LSI",
+      "addr": "ZROUWRELHY7S6FXPL23ZAKZP26IQGS57BU77H5D2QXHOZXBYUQRDKK3SHE",
       "comment": "Wallet1",
       "state": {
         "algo": 1000000000000000,
         "onl": 1,
-        "sel": "XGIljovdG8IjSQJRcvc/vi6UXXTIddUgjZACWZPW23M=",
-        "vote": "zccp6OB5EOh21WMV7A7iVg+CitiZeOao+VYbIkNdPgM=",
-        "voteKD": 10000,
-        "voteLst": 3000000
+        "sel": "cVkeYPxaTpiNv/+UqK02CdwBaiKEBlk0NPgGRZHStiA=",
+        "stprf": "bbd+At6LjySR9koII8WqjKd9bPXUbqdlzQDzcMedZJH+S9+72Odnp3gsFz3W32p25WglOTJ+YKqCgNJOurN3vQ==",
+        "vote": "gwp/uuR0MLU6JSzthTPAhVrv1CDZtmwhmJlXeqqChyQ=",
+        "voteKD": 100,
+        "voteLst": 300000
       }
     },
     {
-      "addr": "SKRRLS4D7YQ3SMC5KIZCMME2NRXW3DJMHAFKZM6YEZMIWIO36YVXZZSGMA",
+      "addr": "KKNXZSFUSUXWRL3O6QTVGB7DF3AEZEY2O3MC77LRN34P77DT26TDEV4RGY",
       "comment": "Wallet2",
       "state": {
         "algo": 4000000000000000,
         "onl": 1,
-        "sel": "OGz10aBiWUWYGKkeqsLbAfb99L75R3HXbiRMa5/7zVk=",
-        "vote": "6lVpwKsSYiXMIsCiYOIFfzwsijfXeEHE+FP8YTzgPMs=",
-        "voteKD": 10000,
-        "voteLst": 3000000
+        "sel": "jfp3Xg4YrCShn8UitZnLxe6Vz+XGjooc/MN88Hlkyrk=",
```

Comment:
why test data changing?

Reply:
I regenerated locally, it did not have the more stateproof field that was added some time ago - if worried about the actual other fields changing, I could revert those?